Makefile

```diff
@@ -5,7 +5,7 @@ MF_DOCKER_IMAGE_NAME_PREFIX ?= mainflux
 BUILD_DIR = build
```

Comment:
it is not important for this PR but it would be nice to add here `MF_RELEASE ?= latest`
and then replace
```
--tag=$(MF_DOCKER_IMAGE_NAME_PREFIX)/$(svc):latest\
```
with
```
--tag=$(MF_DOCKER_IMAGE_NAME_PREFIX)/$(svc):$(MF_RELEASE) \
```

Reply:
@mteodor interesting - please open a new issue for this and send the PR